src/lib/components/Login/OnaOauthLogin/OnaOauthLogin.js

```diff
@@ -1,7 +1,6 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import { useOAuthLogin } from '@onaio/gatekeeper/dist/components/login';
-import { AuthorizationGrantType } from '@onaio/gatekeeper';
+import { useOAuthLogin, AuthorizationGrantType } from '@onaio/gatekeeper/dist/components/login';
```

Comment:
It feels like these should be referenced from `@onaio/gatekeeper` package, not a sub-folder of the module. Are we using the most up-to-date version of gatekeeper? If so, and if we cannot reference these from the package root, we should create a ticket in js-tools to update the package to expose these in this way. 🙏

Reply:
@cKellyDesign Have tested with several versions even the latest `V0.0.19` and am getting same results.

Reply:
@ciremusyoka it looks like `useOAuthLogin` is not being imported/exported from the package index. I'll assign a new ticket to you to add that and bump the version so we can avoid referencing the package's dist folder.

cc @moshthepitt

Reply:
Ticket created: https://github.com/onaio/js-tools/issues/150

Reply:
Nice sleuthing!!